src/Mvc/Mvc.ViewFeatures/src/ViewEngines/ViewEngineResult.cs

```diff
@@ -8,20 +8,41 @@
 
 namespace Microsoft.AspNetCore.Mvc.ViewEngines
 {
+    /// <summary>
+    /// Represents the result of a view engine.
+    /// </summary>
     public class ViewEngineResult
     {
         private ViewEngineResult()
         {
         }
 
+        /// <summary>
+        /// The list of locations searched.
+        /// </summary>
         public IEnumerable<string> SearchedLocations { get; private set; }
 
+        /// <summary>
+        /// The <see cref="IView"/>.
+        /// </summary>
         public IView View { get; private set; }
 
+        /// <summary>
+        /// The name of the view.
```

Comment:
```suggestion
        /// Gets or sets the name of the view.
```

Per the style recommendations [here](https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Property).